src/views/dashboard/flow/index.vue

```diff
@@ -0,0 +1,219 @@
+<template lang="pug">
+.query-layout.flow-query-container.query-container
+  .page-header.title-with-subtitle
+    .main-title Flow
+    .subtitle
+      | real-time computation of data streams.
+      a(
+        href="https://docs.greptime.com/user-guide/flow-computation/overview"
+        target="_blank"
+        rel="noopener noreferrer"
+      )
+        | Learn more
+        icon-external-link(style="margin-left: 4px; font-size: 12px")
+  .content-wrapper.query-layout-cards
+    a-card(:bordered="false")
+      template(#title)
+        .results-header
+          span Results
+          span.results-count(v-if="totalResults > 0") 
+            | ({{ totalResults }} {{ totalResults === 1 ? 'record' : 'records' }})
+      template(#extra)
+        a-button(size="small" @click="showCreate") Create Flow
+      DataTable(
+        :data="data"
+        :columns="columns"
+        :loading="loading"
+        :show-context-menu="false"
+      )
+        template(#column-operate="{ record }")
+          a-space
+            a-button(size="small" @click="showEdit(record)") Edit
+            a-popconfirm(content="Confirm to delete?" @ok="del(record)")
```

Comment:
Recommend adding `type="warning"` for the popconfirm. The UI text seems a bit weird, maybe change it to 'Confirm deletion?'